saleor/static/dashboard/js/components/bulk-actions.js

```diff
@@ -1,8 +1,18 @@
-$(document).ready((e) => {
-  function updateCounterSelected() {
-    const count = $('.select-item:checked').length;
-    $('.data-table-header-action-selected-items #counter').text(count);
+function updateCounterSelected () {
+  const count = $('.select-item:checked').length;
+  const $counterTextNode = $('.data-table-header-action-selected-items');
+  let counterText;
+
+  if (count === 1) {
```

Comment:
Django provides API for this: [ngettext](https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#ngettext)